staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -2409,7 +2409,9 @@ func (i *IngressDescriber) describeIngress(ing *networkingv1beta1.Ingress, descr
 		if count == 0 {
 			w.Write(LEVEL_1, "%s\t%s \t%s (%s)\n", "*", "*", backendStringer(def), i.describeBackend(ns, def))
 		}
-		describeIngressAnnotations(w, ing.Annotations)
+
+		// TODO: Move from annotations into Ingress status.
```

Comment:
By the way any idea what this comment refers to?

Reply:
Yeah, that's quite unclear as a todo, please remove?